client/src/reducers/layoutChoice.ts

```diff
@@ -10,23 +10,8 @@ about commonly used names.  Preferentially, pick in the following order:
 
 import type { Action, AnyAction } from "redux";
 import type { RootState } from ".";
-import { EmbeddingSchema, Schema } from "../common/types/schema";
-
-function bestDefaultLayout(layouts: Array<string>): string {
-  const preferredNames = ["umap", "tsne", "pca"];
-  const idx = preferredNames.findIndex((name) => layouts.indexOf(name) !== -1);
-  if (idx !== -1) return preferredNames[idx];
-  return layouts[0];
-}
-
-function setToDefaultLayout(schema: Schema): LayoutChoiceState {
```

Comment:
this function is now split out into `selectAvailableLayouts` and `getCurrentLayout` for reusability